rpm_spec/subpackages/manageiq-appliance

```diff
@@ -65,6 +65,7 @@ Requires: cockpit-ws
 # Add sub uids/gids for running non-privileged containers
 grep manageiq /etc/subuid >/dev/null 2>&1 || /usr/sbin/usermod --add-subuids 100000-165535 manageiq
 grep manageiq /etc/subgid >/dev/null 2>&1 || /usr/sbin/usermod --add-subgids 100000-165535 manageiq
+/usr/sbin/usermod -a -G disk manageiq
```

Comment:
Also, very minor, but for future readers, I wonder if it's easier to use the full option names?

```suggestion
/usr/sbin/usermod --append --groups disk manageiq
```

Reply:
We should check what the command does when the user is already part of the group, if it returns a non-zero exit code that will cause the rpm scriptlet to fail

Reply:
```
[root@manageiq ~]# groups manageiq
manageiq : manageiq
[root@manageiq ~]# /usr/sbin/usermod --append --groups disk manageiq
[root@manageiq ~]# echo $?
0
[root@manageiq ~]# groups manageiq
manageiq : manageiq disk
[root@manageiq ~]# /usr/sbin/usermod --append --groups disk manageiq
[root@manageiq ~]# echo $?
0
```

Seems like it returns `0` in either case, might be a good idea to check if it is in the group first though `groups manageiq | grep disk >/dev/null 2>&1 || /usr/sbin/usermod ...` just so it is more obvious